### ISSUE TITLE
Repeat keycodes not sent

### ISSUE THREAD

**Issue by cmdrkotori** [gnif/LookingGlass#75]
### Required information

Host CPU: AMD FX(tm)-8320 Eight-Core Processor
Host GPU: [AMD/ATI] Curacao XT / Trinidad XT [Radeon R7 370 / R9 270X/370X]
Guest GPU: [AMD/ATI] Cape Verde PRO [Radeon HD 7750/8740 / R7 250E]
Host Kernel version: 4.16.12
Host QEMU version: QEMU emulator version 2.12.0
Using LookingGlass git master on client and host

I first noticed this when typing a password field and making a typo.

Steps to reproduce
1. Open notepad/wordpad
2. Type gibberish
3. Hold down the backspace key

**Comment by r4m0n** [comment]
Here it works just fine, you probably have key repetition disabled somewhere, maybe in the VM?

**Comment by cmdrkotori** [comment]
It works in virt-manager.

**Comment by qqwa** [comment]
I have the same issue, it worked before with alpha10(if i remember correctly), but stopped working with alpha11.

**Comment by gnif** [comment]
Did you update Qemu? Key repeat is normally handled by the virtual keyboard device in Qemu, not in LG

**Comment by cmdrkotori** [comment]
I'm using qemu-patched from AUR which is based on 2.12.0. The problem is also present on vanilla 2.12.0. This is the latest qemu release.

**Comment by cmdrkotori** [comment]
Right, according to your suggestion I tried a different driver

The bug is present with (PS2 keyboard)
```
        qemu-system-x86_64 \
          -ctrl-grab \
          -enable-kvm \
          -cpu host,hv-time,kvm=off \
          -smp cores=4 \
          -m 8G \
          -M q35 \
          -vga none \
          -netdev tap,id=hostnet,ifname=tap1,script=no,downscript=no \
          -net nic,model=virtio,macaddr=MACCENSORED,netdev=hostnet \
          -netdev user,id=usernet,smb=/media/DRIVE-C/tux/vms/share \
          -net nic,model=virtio,macaddr=MACCENSORED,netdev=usernet \
          -usb \
          -device usb-ehci,id=ehci \
          -device usb-host,vendorid=0x24c6,productid=0x543a,bus=usb-bus.0 \
          -device ioh3420,bus=pcie.0,addr=1c,multifunction=on,port=1,chassis=1,id=root.1 \
          -device vfio-pci,host=07:00.0,bus=root.1,addr=00.0,multifunction=on,x-vga=on \
          -device ich9-intel-hda \
          -device hda-micro \
          -drive if=virtio,file=vm/win10/disk.img,media=disk \
          -boot menu=on,splash=splash/boot.jpg,splash-time=5000 \
          -name win10 \
          -device ivshmem-plain,memdev=ivshmem \
          -object memory-backend-file,id=ivshmem,share=on,mem-path=/dev/shm/looking-glass,size=32M \
          -object memory-backend-file,id=mem,size=8G,mem-path=/hugetlbfs,share=on \
          -device virtio-serial-pci \
          -device virtserialport,chardev=spicechannel0,name=com.redhat.spice.0 \
          -chardev spicevmc,id=spicechannel0,name=vdagent \
          -spice unix,addr=/tmp/looking-glass.socket,disable-ticketing
```
The big is NOT present with (USB keyboard)
```
        qemu-system-x86_64 \
          -ctrl-grab \
          -enable-kvm \
          -cpu host,hv-time,kvm=off \
          -smp cores=4 \
          -m 8G \
          -M q35 \
          -vga none \
          -netdev tap,id=hostnet,ifname=tap1,script=no,downscript=no \
          -net nic,model=virtio,macaddr=MACCENSORED,netdev=hostnet \
          -netdev user,id=usernet,smb=/media/DRIVE-C/tux/vms/share \
          -net nic,model=virtio,macaddr=MACCENSORED,netdev=usernet \
          -usb \
          -device usb-ehci,id=ehci \
          -device usb-kbd,bus=usb-bus.0 \
          -device usb-mouse,bus=usb-bus.0 \
          -device usb-host,vendorid=0x24c6,productid=0x543a,bus=usb-bus.0 \
          -device ioh3420,bus=pcie.0,addr=1c,multifunction=on,port=1,chassis=1,id=root.1 \
          -device vfio-pci,host=07:00.0,bus=root.1,addr=00.0,multifunction=on,x-vga=on \
          -device ich9-intel-hda \
          -device hda-micro \
          -drive if=virtio,file=vm/win10/disk.img,media=disk \
          -boot menu=on,splash=splash/boot.jpg,splash-time=5000 \
          -name win10 \
          -device ivshmem-plain,memdev=ivshmem \
          -object memory-backend-file,id=ivshmem,share=on,mem-path=/dev/shm/looking-glass,size=32M \
          -object memory-backend-file,id=mem,size=8G,mem-path=/hugetlbfs,share=on \
          -device virtio-serial-pci \
          -device virtserialport,chardev=spicechannel0,name=com.redhat.spice.0 \
          -chardev spicevmc,id=spicechannel0,name=vdagent \
          -spice unix,addr=/tmp/looking-glass.socket,disable-ticketing
```
i.e.
```
          -device usb-kbd,bus=usb-bus.0 \
          -device usb-mouse,bus=usb-bus.0 \
```
will fix this issue.

The bug is in qemu. Sorry for taking your time.
